fml/platform/fuchsia/message_loop_fuchsia.h

```diff
@@ -28,6 +28,8 @@ class MessageLoopFuchsia : public MessageLoopImpl {
   async::Loop loop_;
   async::Task task_;
 
+  std::mutex task_mutex_;
```

Comment:
Super minor nit: `task_mutex_` before `task_` because the `task_` because you need the mutex before the task. Either way, it doesn't matter too much.

Reply:
+1